src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -71,12 +71,31 @@ public void testCustomizeOptions() {
 
         //no question specific options to test
 
-        ______TS("CONTRIB: set visibility options");
+        ______TS("CONTRIB: verify only valid visibility options are visible");
 
-        feedbackEditPage.enableOtherVisibilityOptionsForNewQuestion();
-        //TODO: click and ensure can see answer for recipients,
-        //giver team members, recipient team members
-        //are always the same. (under visibility options)
+        assertTrue(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_AND_INSTRUCTORS"));
+        assertTrue(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_VISIBLE_TO_INSTRUCTORS"));
+        assertFalse(feedbackEditPage
+                .isVisibilityDropdownOptionHiddenForNewQuestion("ANONYMOUS_TO_RECIPIENT_AND_TEAM_VISIBLE_TO_INSTRUCTORS"));
+        assertFalse(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("VISIBLE_TO_INSTRUCTORS_ONLY"));
+        assertTrue(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("VISIBLE_TO_RECIPIENT_AND_INSTRUCTORS"));
+        assertTrue(feedbackEditPage.isVisibilityDropdownSeparatorHiddenForNewQuestion());
+        assertTrue(feedbackEditPage.isVisibilityDropdownOptionHiddenForNewQuestion("OTHER"));
+
+        ______TS("CONTRIB: verify correct output params for selected visibility option");
+
+        // default (first option)
```

Comment:
I'm trying to follow the existing user experience when you create a new question in an empty session. This is the exact experience (`Please select ...` but default to first option); I think the developers' intention was to emphasise to the user the need to take a look? Over here it similarly gives a cue that the option was not copied from the previous question. If the user really does not want to take a look, it will default to the first option.